test/test_core.py

```diff
@@ -5948,9 +5948,8 @@ def test_istream(self):
       self.set_setting('LINKABLE', linkable)
       self.do_core_test('test_istream.cpp')
 
-  def test_fs_dir_wasmfs(self):
-    self.emcc_args += ['-sWASMFS']
-    self.emcc_args += ['-sFORCE_FILESYSTEM']
+  def test_fs_dir(self):
```

Comment:
Oh I see.. this was a wasmfs-only test?  I wonder if the test can simply be removed?   Do we know why a wasmfs-only tests was even added in the first place?

Presumably it was because the other tests we have that use directories didn't previously pass.. but maybe now we can just run the existing pre-wasmfs tests under wasmfs and just delete this test?

Reply:
Yes I think that would make sense to me.   Assuming this test was just added to test the wasmfs JS API.